mlir/test/Dialect/ArmSME/enable-arm-streaming-invalid.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: mlir-opt %s -enable-arm-streaming="if-contains-scalable-vectors if-required-by-ops" -verify-diagnostics
+// RUN: mlir-opt %s -enable-arm-streaming="if-compatible-and-scalable if-required-by-ops" -verify-diagnostics
```

Comment:
`if-scalable-and-support` would make more sense to me:
* "scalability" is the more prominent feature that we need to identify
* "supported" rather than "compatible" - I'm not sure what "compatibility" would mean in this context? "compatible" with what?

This blog post would use terms like "supported" and "legal"/"illegal". So "legal" is also an option
* https://community.arm.com/arm-community-blogs/b/architectures-and-processors-blog/posts/arm-scalable-matrix-extension-introduction

Reply:
:+1: `if-scalable-and-supported` sounds better :)